crates/worker/src/purgatory.rs

```diff
@@ -116,6 +111,63 @@ impl Inner {
             .push((id, to_purge_timestampe));
     }
 
+    async fn global_state_purge(db_client: DynamoDBClient, s3_client: S3Client) {
+        const SYNC_FROM_OFFSET: Option<Duration> = PURGE_INTERVAL.checked_mul(6);
+
+        let mut global_state = GlobalState::new(db_client.clone(), s3_client.clone());
+        let sync_from = Utc::now() - SYNC_FROM_OFFSET.unwrap();
```

Comment:
can we avoid `unwrap()` here?

Reply:
Why? That it is invariant that it won't fail ever with specified `PURGE_INTERVAL`.